tests/utilities/test_logging.py

```diff
@@ -223,10 +221,13 @@ def test_log_manager_startup_and_shutdown(logger, log_manager):
 
 
 def test_log_manager_batches_logs(logger, log_manager, monkeypatch):
-    monkeypatch.setattr(prefect.utilities.logging, "MAX_BATCH_LOG_LENGTH", 100)
+    monkeypatch.setattr(prefect.utilities.logging, "MAX_BATCH_LOG_SIZE", 1000)
+    monkeypatch.setattr(prefect.utilities.logging, "MAX_LOG_SIZE", 500)
+
     # Fill up log queue with multiple logs exceeding the total batch length
```

Comment:
```suggestion
    # Fill up log queue with multiple logs exceeding the MAX_BATCH_LOG_SIZE
```

10 * 50 = 500, which imho doesn't anymore exceed the max_batch_log_size

Reply:
Each log is ~350 bytes — we can be sure this test is working as intended because we assert logs are sent 5x

```
assert log_manager.client.write_run_logs.call_count == 5
```